CHANGLOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+New Changes should be put in this changelog as `## [Unreleased] - TYPE`, where `TYPE`,
+is one of `MAJOR`, `MINOR`, or `PATCH` respectively.
+The version and date will automatically be inserted when the release workflow runs.
```

Comment:
This is true for repo's that use the changlelog scripts to update this - Make it clear that this is a manually updated CL and remove the references to ## [Unreleased] - TYPE and the automation

Reply:
Ahh yes, in my head it was just going to magically work 😆